docs/user-guide/zonal-average.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>"A zonal average (or zonal mean) is a statistical measure that represents the average of a variable along one or more lines of constant latitude or over a latitudinal band."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>